gimme

```diff
@@ -316,13 +364,16 @@ unset GOOS
 unset GOPATH
 unset GOROOT
 
+GIMME_VERSION_PREFIX="$(_realpath "${GIMME_VERSION_PREFIX}")"
+GIMME_ENV_PREFIX="$(_realpath "${GIMME_ENV_PREFIX}")"
```

Comment:
And I'm using a func version instead of `readlink -f` because of the great GNU v. BSD schism.